airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -159,6 +161,18 @@ def convert_schema_into_query(
 
         return query_template
 
+    @staticmethod
+    def serialize_protobuf_message(message) -> str:
+        """
+        This method tryies to serialize a given object into JSON if the given object is a protocol bugger message.
+        If not, the normal str() is applied. That helps keep the complex objects written as parsable structures.
+        """
+
+        try:
+            return json_format.MessageToJson(message._pb).replace('\n', '')
+        except:
```

Comment:
What is the exact exception type we expect back if this is not json serializable? For this `except` block we should try to target the specific exception that might be thrown so that we don't accidentally swallow unexpected ones

Reply:
I've added the type of AttributeError cause that is what I expect. If there is no "_pb" then fallback to str().